src/revaultd/mod.rs

```diff
@@ -42,6 +43,9 @@ pub struct RevaultD {
 
 impl RevaultD {
     pub fn new(config: &Config) -> Result<RevaultD, RevaultDError> {
+        let span = span!(Level::INFO, "revaultd");
+        let _enter = span.enter();
```

Comment:
nit: why keeping it if you don't use it ?

Reply:
you need to allocate it and then the span will be closed when the var is dropped

Reply:
Ok, would have thought the compiler would figure out the scope by itself